Cargo.toml

```diff
@@ -11,24 +9,42 @@ license = "Apache-2.0"
 repository = "https://github.com/AhaLabs/scaffold-stellar"
 version = "0.0.1"
 
-[workspace.dependencies]
-soroban-sdk = { version = "22.0.7" }
-soroban-token-sdk = { version = "22.0.1" }
-stellar-pausable = { git = "https://github.com/OpenZeppelin/stellar-contracts", tag = "v0.2.0" }
-stellar-pausable-macros = { git = "https://github.com/OpenZeppelin/stellar-contracts", tag = "v0.2.0" }
-stellar-fungible = { git = "https://github.com/OpenZeppelin/stellar-contracts", tag = "v0.2.0" }
+[workspace.dependencies.soroban-sdk]
+version = "22.0.7"
+
+[workspace.dependencies.soroban-token-sdk]
+version = "22.0.1"
+
+[workspace.dependencies.stellar-default-impl-macro]
+git = "https://github.com/OpenZeppelin/stellar-contracts"
+tag = "v0.3.0"
+
+[workspace.dependencies.stellar-fungible]
+git = "https://github.com/OpenZeppelin/stellar-contracts"
+tag = "v0.2.0"
+
+[workspace.dependencies.stellar-non-fungible]
+git = "https://github.com/OpenZeppelin/stellar-contracts"
+tag = "v0.3.0"
+
+[workspace.dependencies.stellar-pausable]
+git = "https://github.com/OpenZeppelin/stellar-contracts"
+tag = "v0.3.0"
+
+[workspace.dependencies.stellar-pausable-macros]
+git = "https://github.com/OpenZeppelin/stellar-contracts"
+tag = "v0.3.0"
 
 [profile.release]
 opt-level = "z"
```

Comment:
@willemneal I got these changes in the release profile after running `stellar scaffold generate` to add the NFT example ... are these changes ok?

Reply:
Yeah I think they are fine! Mostly seems like a reordering and fixing debug to be a boolean.